### PR TITLE
pgjwt: init at 0.0.1

### PR DESCRIPTION
**Pull request by spinus**
###### Motivation for this change

Needed JWT token manipulation inside PostgreSQL (easier work with PostgREST).

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/servers/sql/postgresql/pgjwt/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchgit}:
```
Comment (joachifm) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: Nit: please add some whitespace after the first and before the last brace in the
parameter list and around `=` in the `name` and `version` attributes below.

File: pkgs/servers/sql/postgresql/pgjwt/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+stdenv.mkDerivation rec {
+  name="pgjwt-${version}";
+  version="0.0.1";
+  src = fetchgit {
```
Comment (joachifm) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: Does this rely on submodules? If not, consider using a source archive, e.g., via `fetchFromGitHub`.

Comment (spinus) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: nope, just simple repo.
Sure, I can change that. Does fetchFromGitHub has any advantage over that?

Comment (joachifm) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: @spinus yes, it's more bandwidth efficient and supports caching.

Comment (joachifm) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: (Using source archive that is, `fetchFromGitHub` is just a convenient way of doing it).

File: pkgs/servers/sql/postgresql/pgjwt/default.nix (hunk 1 above)
Comment (spinus) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: I wish we have formatting tool :-) I'll fix

Comment (spinus) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: done

File: pkgs/servers/sql/postgresql/pgjwt/default.nix (hunk 2 above)
Comment (spinus) [inline pkgs/servers/sql/postgresql/pgjwt/default.nix]: done

**Comment by spinus** [comment]
@joachifm I fixed some stuff you mentioned. If you find a moment, please let me know if that is ok now.

**Pull request merged**